src/Sarif.Viewer.VisualStudio/SarifSnapshot.cs

```diff
@@ -0,0 +1,289 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Windows.Forms;
+
+using EnvDTE;
+
+using Microsoft.CodeAnalysis.Sarif.Driver;
+using Microsoft.CodeAnalysis.Sarif.Driver.Sdk;
+using Microsoft.CodeAnalysis.Sarif.Sdk;
+using Microsoft.CodeAnalysis.Sarif.Writers;
+using Microsoft.VisualStudio;
+using Microsoft.VisualStudio.Shell.Interop;
+using Microsoft.VisualStudio.Shell.TableManager;
+using Microsoft.VisualStudio.TextManager.Interop;
+
+namespace SarifViewer
+{
+    class SarifSnapshot : TableEntriesSnapshotBase
+    {
+        private string _projectName;
+        private readonly List<SarifError> _errors;
+        private Dictionary<string, string> _remappedFilePaths;
+        private List<Tuple<string, string>> _remappedPathPrefixes;
+        private Dictionary<string, NewLineIndex> _fileToNewLineIndexMap;
+
+        internal SarifSnapshot(string filePath, IEnumerable<SarifError> errors)
+        {
+            FilePath = filePath;
+            _errors = new List<SarifError>();
+            _errors.AddRange(errors);
+
+            _remappedFilePaths = new Dictionary<string, string>(StringComparer.OrdinalIgnoreCase);
+            _remappedPathPrefixes = new List<Tuple<string, string>>();
+
+            _fileToNewLineIndexMap = new Dictionary<string, NewLineIndex>(StringComparer.OrdinalIgnoreCase);
+        }
+
+        public override int Count
+        {
+            get { return _errors.Count; }
+        }
+
+        public IList<SarifError> Errors
```

Comment:
You can just write
```
public List<SarifError> Errors { get; }
```
... and initialize `Errors` in the ctor, and then you don't need the backing field `_errors` at all.